lineman/app/css/print_scss_includes.rb

```diff
@@ -3,5 +3,5 @@
   puts "@import '#{filename}';" unless filename =~ /settings/
 end
 Dir[ File.join('../components', '**', '*.scss') ].each do |filename|
-  puts "@import '#{filename}';" unless filename =~ /settings/
```

Comment:
No I've looked and both unless ~settings lines are rubbish. Vestigal css stuff. @HSalmon can you remove the other unless too?